services/OneSeo_MetaService.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Craft;
+
+/**
+ * One SEO Meta Service
+ */
+class OneSeo_MetaService extends BaseApplicationComponent
+{
+  public $meta = array(
+    'metaTitle' => [],
+    'metaDescription' => '',
+    'metaImage' => ''
+  );
+
+  public function setMetaTitle($title)
+  {
+    if (is_array($title))
+    {
+      $this->meta['metaTitle'] = $title;
+    }
+    elseif (is_string($title))
+    {
+      $this->meta['metaTitle'][0] = $title;
```

Comment:
is the reason for setting the `0`th element to the string so people can easily push things into the array? like:

```
$this->setMetaTitle('category');
...
$this->setMetaTitle('detail');
```

if so, it would just be overwriting the 0th element every time, right? this might be a bit more readable if it was:

```
if (!is_array($title)) {
  $title = array($title);
}
$this->meta['metaTitle'] = $title;
```